routes/opportunity.js

```diff
@@ -30,6 +30,7 @@ const opportunityManager = new OpportunityManager(),
  *             - SCHOLARSHIP
  *             - CONFERENCE
  *             - CODINGCOMPETITION
+ *        description: Retrieve a list of opportunities based on particular type or only for female.
  *        description: Retrieve a list of opportunities based on particular type.
  *      - in: query
```

Comment:
In documentation also, you should add `female` query parameter, 
Visiting http://localhost:3030/playground/#/default/get_opportunity shall have `female` parameter added.